src/main/java/org/openhds/service/impl/census/LocationHierarchyLevelService.java

```diff
@@ -19,9 +19,10 @@ public LocationHierarchyLevelService(LocationHierarchyLevelRepository locationHi
     }
 
     @Override
-    protected LocationHierarchyLevel makeUnknownEntity() {
+    public LocationHierarchyLevel makePlaceHolder(String id, String name) {
         LocationHierarchyLevel locationHierarchyLevel = new LocationHierarchyLevel();
-        locationHierarchyLevel.setName("unknown");
+        locationHierarchyLevel.setUuid(id);
+        locationHierarchyLevel.setName(name);
         locationHierarchyLevel.setKeyIdentifier(-1);
```

Comment:
How about `locationHierarchyLevel.setKeyIdentifier(id.hashCode());` so that it can be unique?